Ginger/GingerCoreCommon/Repository/ApplicationModelLib/AppModelParameter.cs

```diff
@@ -132,4 +132,19 @@ public string GetDefaultValue()
 
 
     }
+
+    /// <summary>
+    /// This class is used in the Export Process
+    /// </summary>
+    public class AppParameters
+    {     
+        public string ItemName { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public ObservableList<OptionalValue> OptionalValuesList = new ObservableList<OptionalValue>();
```

Comment:
You not goinf to save it to our repository XML so for what the [IsSerializedForLocalRepository] attribute?

Reply:
Done